climada/hazard/tc_tracks.py

```diff
@@ -348,7 +348,7 @@ def read_ibtracs_netcdf(self, provider=None, rescale_windspeeds=True, storm_id=N
         storm_id : str or list of str, optional
             IBTrACS ID of the storm, e.g. 1988234N13299, [1988234N13299, 1989260N11316].
         year_range : tuple (min_year, max_year), optional
-            Year range to filter track selection. Default: (1980, 2018)
+            Year range to filter track selection. Default: None.
```

Comment:
If find it extremely disturbing that there is a default year range. I didn't even know that this exists even though I'm using this function regularly. So, if I specify only a basin like 'NA', there will be no tracks for the year 2019 (for example). This is totally counter-intuitive and I propose to remove the default year range.